geomesa-convert/geomesa-convert-common/pom.xml

```diff
@@ -36,6 +40,14 @@
             <artifactId>geomesa-utils</artifactId>
         </dependency>
         <dependency>
+            <groupId>javax.media</groupId>
+            <artifactId>jai_core</artifactId>
```

Comment:
They end up scope provided, same as in the other projects. Actually, I might not need them anymore... I'll double check that, and also make sure the dependencies haven't changed.